README.md

```diff
@@ -80,204 +80,16 @@ spec:
-  imageName: postgres
-  dbName: db-demo
-EOS
-```
-
-#### Set labels on the application
-
-Now the we need to set arbitrary labels on the application's `DeploymentConfig` in order for the Service Binding Operator to be able to find the application.
-
-The labels are:
-
-* `connects-to=postgres` - indicates that the application needs to connect to a PostgreSQL DB
-* `environment=demo` - indicates the demo environment - it narrows the search
-
-``` bash
- kubectl patch dc nodejs-app -p '{"metadata": {"labels": {"connects-to": "postgres", "environment":"demo"}}}'
-```
-
-#### Express an intent to bind the DB and the application
-
-Now the only thing that remains is to connect the DB and the application. We let the Service Binding Operator to 'magically' do the connection for us.
-
-Create the following `ServiceBindingRequest`:
-
-``` bash
-cat <<EOS |kubectl apply -f -
----
-apiVersion: apps.openshift.io/v1alpha1
-kind: ServiceBindingRequest
-metadata:
-  name: binding-request
-  namespace: service-binding-demo
-spec:
-  applicationSelector:
-    matchLabels:
-      connects-to: postgres
-      environment: demo
-    resourceKind: DeploymentConfig
-  backingServiceSelector:
-    resourceKind: postgresql.baiju.dev
-    resourceVersion: v1alpha1
-    resourceRef: db-demo
-  mountPathPrefix: “”
-EOS
-```
-
-There are 2 parts in the request:
-
-* `applicationSelector` - used to search for the application based on the labels that we set earlier and the `resourceKind` of the application to be a `DeploymentConfig`.
-* `backingServiceSelector` - used to find the backing service - our operator-backed DB instance called `db-demo`.
-
-That causes the application to be re-deployed.
-
-Once the new version is up, go the application's route to check the UI. In the header you can see `(DB: db-demo)` which indicates that the application is connected to a DB and its name is `db-demo`. Now you can try the UI again but now it works!
-
-When the `ServiceBindingRequest` was created the Service Binding Operator's controller injected the DB connection information as specified in the OLM descriptor below, into the
-application's `DeploymentConfig` as environment variables via an intermediate `Secret` called `binding-request`:
-
-``` yaml
-spec:
-  template:
-    spec:
-      containers:
-        - envFrom:
-          - secretRef:
-              name: binding-request
-```
-
-#### ServiceBindingRequestStatus
-
-`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status
-
-| Field | Description |
-|-------|-------------|
-| BindingStatus | The binding status of Service Binding Request |
-| Secret | The name of the intermediate secret |
-
-That's it, folks!
+[Binding an Imported app to an In-cluster Operator Managed PostgreSQL Database](examples/bind_imported_app_to_incluster_operator_managed_PostgreSQL_db/README.md)
```

Comment:
There's the scenario which @pmacik added, let's list that too?

Reply:
Thx - just added it!